server/app/controllers/FileController.java

```diff
@@ -64,13 +71,38 @@ public CompletionStage<Result> show(Request request, long applicantId, String fi
             });
   }
 
+  private long extractApplicantIdFromFileKey(String fileKey) {
+    String applicantString = fileKey.split("/")[0];
+    long applicantid = (long) Long.parseLong(applicantString.split("-")[1].trim());
+    return applicantid;
+  }
+
```

Comment:
issue (blocking): the file key has the *applicant* ID embedded in it, not the application ID https://github.com/seattle-uat/civiform/blob/main/server/app/services/cloud/FileNameFormatter.java#L35